### PR TITLE
Added JQuery cheat sheet

### PR DESCRIPTION
**Pull request by epik**
# Goodie Cheat Sheet Pull Request Template

**What does your Instant Answer do?**

Cheat sheet for JQuery

**What is the data source for your Instant Answer? (Provide a link if possible)**

http://api.jquery.com/

**Why did you choose this data source?**

It is the official documentation for JQuery

**Are there any other alternative (better) data sources?**

No

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

Javascript programmers and web developers

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

No

**Are you having any problems? Do you need our help with anything?**

No

**Where did you hear about DuckDuckHack? (For first time contributors)**

Google

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

screenshot 2015-09-01 21 44 46

-----
IA Page: https://duck.co/ia/view/jquery_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@epik thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: share/goodie/cheat_sheets/json/jquery.json [hunk 1]
```diff
@@ -0,0 +1,254 @@
+{
+    "id": "jquery_cheat_sheet",
+    "name": "JQuery",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/jquery.json]: nit: Should be formatted as `jQuery` (lowercase j)

File: share/goodie/cheat_sheets/json/jquery.json [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+    "description": "JavaScript Library",
+    "metadata": {
+        "sourceName": "JQueryAPI",
+        "sourceUrl": "http://api.jquery.com/"
+    },
+    "template_type": "terminal",
+    "section_order": [
+        "Selectors",
+		"Traversing",
+		"Css",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/jquery.json]: Should be formatted as `CSS`

File: share/goodie/cheat_sheets/json/jquery.json [hunk 3]
```diff
@@ -0,0 +1,254 @@
+{
+    "id": "jquery_cheat_sheet",
+    "name": "jQuery",
+    "description": "JavaScript Library",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/jquery.json]: Let's remove this, it's a little redundant.

File: share/goodie/cheat_sheets/json/jquery.json [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+    "metadata": {
+        "sourceName": "jQueryAPI",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/jquery.json]: Let's use `jQuery API Documentation`

File: share/goodie/cheat_sheets/json/jquery.json [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+            "val": "Load data from the server using a HTTP POST request",
+            "key": "$.post(url, data, success, dataType)"
+        }, {
+            "val": "Encode a form's elements as a string for submission",
+            "key": ".serialize()"
+        }, {
+            "val": "Encode a form's elements as an array of names and values",
+            "key": ".serializeArray()"
+        }],
+		"Utilities": [{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/jquery.json]: Whoops, indentation is a little off here

**Comment by moollaza** [comment]
@epik this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@epik congrats! It's live: https://duckduckgo.com/?q=jquery+cheat+sheet&t=ffcm&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by epik** [comment]
Awesome.  I am in the process of switching jobs so I will be radio silent
for a couple weeks, but the EnergyStar project is still progressing slowly.

**Pull request merged**